db/migrate/20210518143026_add_token_inheritance.rb

```diff
@@ -0,0 +1,10 @@
+class AddTokenInheritance < ActiveRecord::Migration[6.1]
+  def change
+    change_table :lead_provider_api_tokens, bulk: true do |t|
+      t.change_null :lead_provider_id, true
```

Comment:
If we want STI this needs to be nullable - cause other token classes will not necessarily have a lead provider for them (say, engage and learn token)